docs/existing-aws-infrastructure.md

```diff
@@ -79,14 +76,17 @@ spec:
     id: subnet-0a3507a5ad2c5c8c3
 ```
 
-To tell Cluster API that an EC2 instance should be placed in a particular AZ but allow Cluster API to select which subnet in that AZ can be used, add this to the AWSMachine specification:
+When using MachineDeployments, users can control subnet selection by adding information to the AWSMachineTemplate associated with that MachineDeployment, like this:
 
 ```yaml
 spec:
-  availabilityZone: "us-west-2a"
+  template:
+    spec:
+      subnet:
+        id: subnet-0a3507a5ad2c5c8c3
 ```
 
-These two settings are mutually exclusive. You may use only one or the other, not both.
+Users may either specify `failureDomain` on the Machine or MachineDeployment objects, _or_ users may explicitly specify subnet IDs on the AWSMachine or AWSMachineTemplate objects. The two settings are mutually exclusive.
```

Comment:
```suggestion
Users may either specify `failureDomain` on the Machine or MachineDeployment objects, _or_ users may explicitly specify subnet IDs on the AWSMachine or AWSMachineTemplate objects. If both are specified, the subnet ID is used and the `failureDomain` is ignored.
```

We aren't able to validate cross-object, so we can't prevent a determined user from setting up the scenario where both are specified. Better to be explicit about what happens in that case IMO.

Reply:
This suggested change is included in 1667e3a.